Sources/Server/AppServer.swift

```diff
@@ -130,61 +136,33 @@ public class AppServer {
     self.requests = requests
 
     requests.forEach {
-      let requestedPath = $0.path.joined(separator: "/")
       let requestedResponse = $0.requestedResponse
 
       application?
-        .on($0.method.vaporMethod, $0.vaporParameter) { [unowned self] request -> EventLoopFuture<ClientResponse> in
-          // This property is force-unwrapped because it can never fail,
-          // since the raw value passed is an identical copy of SwiftNIO's `HTTPMethod`.
-          let httpMethod = HTTPMethod(rawValue: request.method.rawValue)!
-          let receivedRequestTimeStamp = Date().timeIntervalSince1970
-
+        .on($0.method.vaporMethod, $0.vaporParameter) { request -> EventLoopFuture<ClientResponse> in
```

Comment:
```suggestion
        .on($0.method.vaporMethod, $0.vaporParameter) { [weak self] request -> EventLoopFuture<ClientResponse> in
```
Is it possible that the request has a strong reference?